src/Symfony/Component/Security/Core/Authorization/Strategy/AbstractDecideStrategy.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Symfony\Component\Security\Core\Authorization\Strategy;
+
+abstract class AbstractDecideStrategy
```

Comment:
As each "stategy" is an implementation of `AccessDecisionManagerInterface` now, I think we should call this class `AbstractAccessDecisionManager` and let it implement the interface. I think we should also rename/remove the `Strategy` namespace.